src/main/antlr/StitchingDSL.g4

```diff
@@ -43,9 +43,13 @@ fieldMappingDefinition : 'renamed from' name ('.'name)?;
 //
 // hydration
 
-underlyingServiceHydration: 'hydrated from' serviceName '.' (syntheticField '.')? topLevelField remoteCallDefinition? objectIdentifier? batchSize?;
+underlyingServiceHydration: 'hydrated from' serviceName '.' (syntheticField '.')? topLevelField remoteCallDefinition? objectResolution? batchSize?;
 
-objectIdentifier: 'object identified by' name;
+objectResolution: (objectByIdentifier | objectByIndex);
+
+objectByIdentifier: 'object identified by' name;
+
+objectByIndex: 'object indexed';
```

Comment:
Naming is hard

`hydrated from UserService.usersByIds(ids: $source.authorIds) using indexes, batch size 5`

`hydrated from UserService.usersByIds(ids: $source.authorIds) using indexation, batch size 5`

I mean I dont hate the `object indexed` and it has symmetry to the other syntax.

But I feel like this read better - thoughts

Reply:
I'm happy with either. @andimarek thoughts?

Reply:
@bbakerman I've updated using your suggested syntax, I prefer `indexes` over `indexation` personally.